test/Model.js

```diff
@@ -3268,6 +3268,35 @@ describe('Model', function () {
     });
   });
 
+  it('Should allow for originalItem on multiple models', (done) => {
```

Comment:
This test fails without your changes you made, correct?

Reply:
Correct